foundations/javascript_basics/problem_solving.md

```diff
@@ -79,7 +79,7 @@ Does your program have an interface? What will it look like?
 Our FizzBuzz solution will be a command line program, so we don't need an interface. The only user interaction will be allowing users to enter a number.
```

Comment:
Since we're changing command line to prompt, this needs to be changed as well:
```suggestion
Our FizzBuzz solution will be a browser console program, so we don't need an interface. The only user interaction will be allowing users to enter a number.
```